devops/Makefile

```diff
@@ -316,9 +329,21 @@ init-patroni:
 	@echo "++\n Init Patroni in $(NAMESPACE) \n++"
 	@oc -n $(NAMESPACE) process -f openshift/patroni-pre-req.yml | oc -n $(NAMESPACE) apply -f -
 
+init-redis:
+	@echo "++\n Init Redis in $(NAMESPACE) \n++"
+	@oc -n $(NAMESPACE) process -f openshift/redis-secrets.yml | oc -n $(NAMESPACE) apply -f -
+
 deploy-patroni:
 	@oc -n $(NAMESPACE) process -f openshift/patroni-deploy.yml -p BUILD_NAMESPACE=$(BUILD_NAMESPACE) -p IMAGE_STREAM_TAG=patroni:12-latest | oc -n $(NAMESPACE) apply -f -
 
+deploy-redis:
+	@oc -n $(NAMESPACE) process -f openshift/redis-ha-deploy.yml -p IMAGE_NAME="redis" -p IMAGE_TAG="6.0.9-alpine" -p IMAGE_REGISTRY="artifacts.developer.gov.bc.ca/docker-remote" | oc -n $(NAMESPACE) apply -f -
```

Comment:
We also have version `6.2.7-alpine` available. Is it not possible to use it?
Because we are using `docker-remote`, can we go for some version 7 maybe?

Reply:
Fair enough for `redis-7`. How about the `6.2.7-alpine`?

Reply:
`6.2.7-alpine` is deployed to OpenShift and deploy files updated.